### ISSUE TITLE
IOS15 sounds stop playing but no errors reports

### ISSUE THREAD

**Issue by rjsheldon** [goldfire/howler.js#1609]
### The Problem

At any point, the sounds on mobile ios15 stop working. The context says it is playing, there's no errors reported but the sounds are all inaudible. Furthermore, new sounds dont play

### Reproduction Steps

just play sounds for a minute or so on a mobile and then they just stop!

### Context

Trying to write a game with multiple m4a files. Tried using an audiosprite but now loading and playing files one at a time but makes no difference

### Howler.js Version

2.2.3

### Affected Browser(s)/Versiuon(s)

safari

**Comment by SalmanMalyk** [comment]
It's also happening on my site too

**Comment by SwenRanj** [comment]
Hi, we are also experiencing this issue on a website that uses canvas and sounds.

We were actually using [SoundJS](https://www.createjs.com/soundjs) when this issue surfaced for us. As we were exploring the issue we also tried HowlerJS to see if the same issue shows up.
We also tried using 'vanilla' HTML audio element, but same issue.

For us, sound is playing correctly, but after a certain time it 'just' stops outputting sound. Audio seems to be playing correctly under the hood, but no sound is outputted.
The issue seems to surface when the website is under heavy load (a lot of canvas drawing) or just after a certain time being on the same website page (mostly after a minute).

Any findings? Ideas?

**Comment by rjsheldon** [comment]
yes, its definitely NOT a howler issue as such. Its the canvas issue on ios
15.X . I tried the latest ios15.6 and its the same.

I reduced the load on the canvas by reducing the number of animations on
screen and it all works fine now .

Also, if you change from canvas to WebGL that also fixes it.

**Comment by The-Linguist** [comment]
It looks like we have to wait for Safari 17.0 or 18.0 and just hope that engineers who work for Apple care.

**Comment by The-Linguist** [comment]
According to my tests, Safaris on different iPads are like different cousin Safaris responding differently.
